Tools/TestWebKitAPI/Tests/WebCore/cocoa/TestGraphicsContextGLCocoa.mm

```diff
@@ -356,6 +384,35 @@ static bool hasMultipleGPUs()
     EXPECT_TRUE(memoryFootprintChangedBy(lastFootprint, footprintChange, footprintError));
 }
 
+// Tests that prepareForDisplayWithFinishedSignal submits the underlying fence.
+// E.g. no other work will be done to context after prepare, but the finished signal will
+// eventually come.
+TEST_P(AnyBackendAPITest, FinishIsSignaled)
+{
+    auto context = createDefaultTestContext({ 2048, 2048 });
+    ASSERT_NE(context, nullptr);
+    context->markContextChanged();
+    context->clearColor(0.f, 1.f, 0.f, 1.f);
+    context->clear(WebCore::GraphicsContextGL::COLOR_BUFFER_BIT);
+    std::atomic<bool> gotSignal = false;
```

Comment:
`signalled`? (`gotSignal` sounds horrible to my ear) `while not signalled` reads better than `while not got signal`.